### PR TITLE
Gitpod

### PR DESCRIPTION
**Pull request by marwin1991**
@angrymeir What do you think about GitPod? I often work on different machines so gitpod for my use case would be helpful to contribute :)

### CODE REVIEW

**Comment by angrymeir** [comment]
@marwin1991 I have never used GitPod before, but I'm certainly open to try it!

However, I think the setup is missing dependency installation correct? (As simple as `poetry install`)

**Comment by marwin1991** [comment]
@angrymeir now it should be better :)

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full:latest
```
Comment (angrymeir) [inline .gitpod.Dockerfile:1]: Would it make sense to use the python3.9 workspace image? (Ref: https://hub.docker.com/r/gitpod/workspace-python-3.9)

**Review by angrymeir — COMMENTED** [review]
🙂

File: .gitpod.Dockerfile (hunk 1 above)
Comment (marwin1991) [reply .gitpod.Dockerfile:1]: What about  https://hub.docker.com/r/gitpod/workspace-python ?

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+image:
+  file: .gitpod.Dockerfile
+  
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - init: poetry install # runs during prebuild
+    command: echo 'poetry install finished'
+
+# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
+ports:
```
[the remark this answers is not in the document]
Comment (marwin1991) [reply .gitpod.yml]: Fixed

File: .gitpod.Dockerfile (hunk 1 above)
Comment (angrymeir) [reply .gitpod.Dockerfile:1]: I would prefer to take the fixed version [1], as it provides us with less uncertainty about which python version is used. Image [2] currently runs python3.8 and I could not find information on how one gets notified about when they change interpreter version...

[1] https://hub.docker.com/r/gitpod/workspace-python-3.9
[2] https://hub.docker.com/r/gitpod/workspace-python

Comment (marwin1991) [reply .gitpod.Dockerfile]: Done

**Pull request merged**